torch/fx/graph_module.py

```diff
@@ -671,10 +671,9 @@ def _gen_model_repr(module_name: str, module: torch.nn.Module) -> Optional[str]:
                 blobified_modules.append(module_name)
                 module_repr = module.__repr__().replace("\r", " ").replace("\n", " ")
                 # weights_only=False as this is legacy code that saves the model
-                module_str = (
-                    f"torch.load(r'{module_file}', weights_only=False) # {module_repr}"
-                )
-            model_str += f"{tab * 2}self.{module_name} = {module_str}\n"
+                model_str += f"{tab * 2}setattr(self, '{module_name}', torch.load(r'{module_file}', weights_only=False)) # {module_repr}\n"
+            else:
+                model_str += f"{tab * 2}setattr(self, '{module_name}', {module_str})\n"
```

Comment:
It would be nice if we could preserve the direct syntax when the attr name is legal